frontend/app-config.json

```diff
@@ -405,7 +406,8 @@
           "exchangeUrl": "https://app.pangolin.exchange/#/swap?inputCurrency=0x483416eb3afa601b9c6385f63cec0c82b6abf1fb&outputCurrency=0xc7198437980c041c805a1edcba50c1ce5db95118",
           "fightGas": "0.006"
         }
-      }
+      },
+      "pvpSupportedChains": ["HECO","OEC"]
```

Comment:
Let me know which chains will be supported on prod (don't worry, there's still flag in place, so it won't be available, but we can prepare for day 1)